wpilibj/src/athena/java/edu/wpi/first/wpilibj/SensorBase.java

```diff
@@ -89,10 +89,10 @@ public static void setDefaultSolenoidModule(final int moduleNumber) {
   protected static void checkSolenoidModule(final int moduleNumber) {
     if (!SolenoidJNI.checkSolenoidModule(moduleNumber)) {
       StringBuilder buf = new StringBuilder();
-      buf.append("Requested solenoid module is out of range. Minimumm: 0, Maximum: ");
-      buf.append(kPCMModules);
-      buf.append(", Requested: ");
-      buf.append(moduleNumber);
+      buf.append("Requested solenoid module is out of range. Minimumm: 0, Maximum: ")
+        .append(kPCMModules)
+        .append(", Requested: ")
+        .append(moduleNumber);
```

Comment:
These are functionally equivalent, but one or the other just seems stylistic.  Does PMD actually require this approach?